src/annotator/guest.js

```diff
@@ -442,12 +444,14 @@ export default class Guest extends Delegator {
      */
     const highlight = anchor => {
       const range = resolveAnchor(anchor);
+      const colorList = ['#f0c1b3', '#acc5c8', '#93b8d3', '#eed8ae']
       if (!range) {
         return anchor;
       }
 
+		//also pass it the color to set it to, where get the color?
       const highlights = /** @type {AnnotationHighlight[]} */ (highlightRange(
-        range
+        range, Math.random() * (6 - 1) + 1
```

Comment:
Instead of passing in an integer corresponding to one color in a fixed list in the css, and then having to make a new css class for each color, why not pass in the color directly and set it in the element style? So here you'd pass in something like `colorList[Math.random() * colorList.length]`, and then adding colors takes changing one place.